app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -108,10 +114,17 @@ invitationForm = React.createClass
         @errorFor fieldName
 
   errorFor: (fieldName) ->
-    @props.fields[fieldName].error
+    if fieldName == 'domainIds'
```

Comment:
Zie ik niet. Je krijgt volgens mij de exact zelfde 2 if-statements, maar dan met een check op null, wat een duidelijke edge case is.